### DISCUSSION TITLE
Raspberry Pi Zero 2?

### DISCUSSION THREAD

**Discussion by cognitivegears** [gtoal/pitrex#6] [Q&A]
Wanted to check whether anyone had any experience with trying the Pitrex with the Raspberry Pi Zero 2? My Pitrex is supposed to be here soon, and as luck has it I just managed to snag a Zero 2, and I'm hoping I can use that with the Pitrex. Thoughts?

p.s. thanks for such an amazing project for the Vectrex!

**Comment by gtoal** [comment]
You won't be able to use the Zero2 with the PiTrex at the moment.  Malban is working on a port to the new CPU.  (Also even when we get it working, you'll need to solder a header to the Zero2 which doesn't yet come with one attached).

(PS Best place to keep on top of PiTrex announcements is the Facebook group at https://www.facebook.com/groups/vectrex/ )

**Reply by gtoal** [reply]
(Although a year later... the Zero2 is now supported on the baremetal release; not yet on the linux release)
